db/dcrpg/pgblockchain.go

```diff
@@ -5838,13 +5840,18 @@ func (pgb *ChainDB) GetExplorerTx(txid string) *exptypes.TxInfo {
 		} else {
 			opTAdd = strings.HasPrefix(vout.ScriptPubKey.Asm, "OP_TADD")
 		}
+		// Get a consistent script class string from dbtypes.ScriptClass. Alt.
+		// to using NewScriptClassFromString, decode Hex or use
+		// msgTx.TxOut[i].PkScript, get stdscript.ScriptType, convert with
+		// dbtypes.NewScriptClass
+		dbScriptClass := dbtypes.NewScriptClassFromString(vout.ScriptPubKey.Type)
```

Comment:
One of many things in this changeset that's still not right.  Need to do what's described in the comment and decode the pkScript in the msgTx